AsyncDisplayKit/ASTextNode.h

```diff
@@ -12,6 +12,14 @@
 @protocol ASTextNodeDelegate;
 
 /**
+ * Alignment
+ */
+typedef NS_ENUM(NSUInteger, ASTextNodeVerticalAlignment) {
+    ASTextNodeVerticalAlignmentDefault = 0,
```

Comment:
Could you add a comment here describing what the Default behavior actually is?  In fact, it's probably better to rename this to a description of its behavior (e.g. ...AlignmentBottom = 0,  // Default)